static/js/telescope_availability_app.vue

```diff
@@ -0,0 +1,31 @@
+<template>
```

Comment:
Both of those have javascript that has nothing to do with vue.js in them as well. But it's not a big deal. Can you remove the _app from all the files, we don't do that anywhere else.